docs/tutorial/process-model.md

```diff
@@ -206,3 +213,4 @@ This feature is incredibly useful for two main purposes:
 [context-isolation]: ./context-isolation.md
 [context-bridge]: ../api/context-bridge.md
 [ipcRenderer]: ../api/ipc-renderer.md
+[tutorial]: ./tutorial-prerequisites.md
```

Comment:
```suggestion
[tutorial]: ./tutorial-prerequisites.md

```
To fix lint error

Reply:
Thanks!
I'm going to change this to draft because @erickzhao did some comments in electron/electronjs.org-new#124